.github/workflows/land-blocking.yml

```diff
@@ -71,12 +71,23 @@ jobs:
         run: |
           export CTI_OUTPUT_LOG=$(mktemp)
           echo "::set-env name=CTI_OUTPUT_LOG::$CTI_OUTPUT_LOG"
-          JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
+          if [[ ${{ secrets.ACTIONS_LBT_USE_K8S }} == true ]]; then
             ./scripts/cti \
-            --marker land \
-            --tag ${TEST_TAG} \
-            --report report.json \
-            --run bench
+              --k8s \
+              --tag ${TEST_TAG} \
+              --report report.json \
+              --run bench \
+              --k8s-fullnodes-per-validator=1 \
+              --k8s-num-validators=30 \
+              --emit-to-validator=false
+          else
+            JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
+              ./scripts/cti \
+              --marker land \
+              --tag ${TEST_TAG} \
+              --report report.json \
+              --run bench
```

Comment:
It looks like we don't need this anymore. Any reason to keep it around?

Reply:
I'm keeping this around in case we have to revert back. If everything looks good for a week, we can get rid of this code and the cluster.